src/euth/euth/static/js/Packages/Euth/IdeaCollection/Context/Module.ts

```diff
@@ -0,0 +1,39 @@
+import * as AdhEmbedModule from "../../../Embed/Module";
+import * as AdhPermissionsModule from "../../../Permissions/Module";
+import * as AdhResourceAreaModule from "../../../ResourceArea/Module";
+import * as AdhTopLevelStateModule from "../../../TopLevelState/Module";
+
+import * as AdhEuthWorkbenchModule from "../Workbench/Module";
+
+import * as AdhEmbed from "../../../Embed/Embed";
+import * as AdhResourceArea from "../../../ResourceArea/ResourceArea";
+
+import * as AdhEuthWorkbench from "../Workbench/Workbench";
+
+import RIPeuthProcess from "../../../../Resources_/adhocracy_euth/resources/idea_collection/IProcess";
```

Comment:
No, its just a convention. R/S is short for Resource/Sheet. For example, there is both an `IComment` resource and an `IComment` sheet. So we differentiate by calling them `RIComment`/`SIComment`. Feel free to add some documentation if you think that helps.